olive/passes/onnx/conversion.py

```diff
@@ -319,73 +321,77 @@ def _load_pytorch_model(
                 - the onnx model must be quantized using OnnxBnb4Quantization pass after conversion
         Model attributes is None if the output model should inherit the model attributes from the input model.
         """
+        pytorch_model = None
+        model_attributes = deepcopy(model.model_attributes or {})
         if not model.is_model_loaded_from_hf_config() or not model.hf_config.from_pretrained_args:
             # if the model is not loaded from hf config, or the model loading args is not specified,
             # we can load the model directly
-            return model.load_model(), None
```

Comment:
only changes are: 
- code is indented
- `new_model_attibutes -> model_attributes`
- return is at the end